syntax = "proto2";

package oneflow.xrt;

import "oneflow/xrt/platform.proto";

message XrtClusteringOptions {
  optional int32 minimum_nodes = 1 [default = 1];
  optional int32 maximum_nodes = 2 [default = 1000];
  optional bool strict_clustering = 3 [default = true];
}

message XrtExecutionOptions {
  optional XrtDevice device = 1 [default = CPU_X86];
  optional XrtEngine engine = 2 [default = XLA];
}
